<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>setting</class>
 <widget class="QMainWindow" name="setting">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1124</width>
    <height>855</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>เลือกมื้อของยา</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color: rgb(217, 244, 255)</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QLabel" name="setting_label">
    <property name="geometry">
     <rect>
      <x>360</x>
      <y>50</y>
      <width>431</width>
      <height>71</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>24</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(255, 255, 255);</string>
    </property>
    <property name="frameShape">
     <enum>QFrame::Box</enum>
    </property>
    <property name="lineWidth">
     <number>1</number>
    </property>
    <property name="text">
     <string>   เลือกมื้อของยา</string>
    </property>
    <property name="textFormat">
     <enum>Qt::AutoText</enum>
    </property>
    <property name="scaledContents">
     <bool>false</bool>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
    <property name="wordWrap">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QLabel" name="setting_icon_label">
    <property name="geometry">
     <rect>
      <x>390</x>
      <y>70</y>
      <width>41</width>
      <height>41</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>24</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(255, 255, 255);</string>
    </property>
    <property name="frameShape">
     <enum>QFrame::NoFrame</enum>
    </property>
    <property name="lineWidth">
     <number>1</number>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="textFormat">
     <enum>Qt::AutoText</enum>
    </property>
    <property name="pixmap">
     <pixmap resource="../Resources/resources.qrc">:/icons/setting_icon.png</pixmap>
    </property>
    <property name="scaledContents">
     <bool>true</bool>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
    <property name="wordWrap">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="Line" name="line">
    <property name="geometry">
     <rect>
      <x>-20</x>
      <y>180</y>
      <width>1201</width>
      <height>16</height>
     </rect>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Plain</enum>
    </property>
    <property name="lineWidth">
     <number>3</number>
    </property>
    <property name="orientation">
     <enum>Qt::Horizontal</enum>
    </property>
   </widget>
   <widget class="QCheckBox" name="bb_checkBox">
    <property name="geometry">
     <rect>
      <x>60</x>
      <y>330</y>
      <width>61</width>
      <height>71</height>
     </rect>
    </property>
    <property name="sizePolicy">
     <sizepolicy hsizetype="Minimum" vsizetype="Fixed">
      <horstretch>0</horstretch>
      <verstretch>0</verstretch>
     </sizepolicy>
    </property>
    <property name="font">
     <font>
      <pointsize>14</pointsize>
     </font>
    </property>
    <property name="focusPolicy">
     <enum>Qt::WheelFocus</enum>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="tristate">
     <bool>false</bool>
    </property>
   </widget>
   <widget class="QCheckBox" name="ab_checkBox">
    <property name="geometry">
     <rect>
      <x>820</x>
      <y>480</y>
      <width>61</width>
      <height>71</height>
     </rect>
    </property>
    <property name="sizePolicy">
     <sizepolicy hsizetype="Minimum" vsizetype="Fixed">
      <horstretch>0</horstretch>
      <verstretch>0</verstretch>
     </sizepolicy>
    </property>
    <property name="font">
     <font>
      <pointsize>14</pointsize>
     </font>
    </property>
    <property name="focusPolicy">
     <enum>Qt::WheelFocus</enum>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="tristate">
     <bool>false</bool>
    </property>
   </widget>
   <widget class="QCheckBox" name="bl_checkBox">
    <property name="geometry">
     <rect>
      <x>820</x>
      <y>330</y>
      <width>61</width>
      <height>71</height>
     </rect>
    </property>
    <property name="sizePolicy">
     <sizepolicy hsizetype="Minimum" vsizetype="Fixed">
      <horstretch>0</horstretch>
      <verstretch>0</verstretch>
     </sizepolicy>
    </property>
    <property name="font">
     <font>
      <pointsize>14</pointsize>
     </font>
    </property>
    <property name="focusPolicy">
     <enum>Qt::WheelFocus</enum>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="tristate">
     <bool>false</bool>
    </property>
   </widget>
   <widget class="QCheckBox" name="al_checkBox">
    <property name="geometry">
     <rect>
      <x>440</x>
      <y>480</y>
      <width>61</width>
      <height>71</height>
     </rect>
    </property>
    <property name="sizePolicy">
     <sizepolicy hsizetype="Minimum" vsizetype="Fixed">
      <horstretch>0</horstretch>
      <verstretch>0</verstretch>
     </sizepolicy>
    </property>
    <property name="font">
     <font>
      <pointsize>14</pointsize>
     </font>
    </property>
    <property name="focusPolicy">
     <enum>Qt::WheelFocus</enum>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="tristate">
     <bool>false</bool>
    </property>
   </widget>
   <widget class="QCheckBox" name="bd_checkBox">
    <property name="geometry">
     <rect>
      <x>440</x>
      <y>330</y>
      <width>61</width>
      <height>71</height>
     </rect>
    </property>
    <property name="sizePolicy">
     <sizepolicy hsizetype="Minimum" vsizetype="Fixed">
      <horstretch>0</horstretch>
      <verstretch>0</verstretch>
     </sizepolicy>
    </property>
    <property name="font">
     <font>
      <pointsize>14</pointsize>
     </font>
    </property>
    <property name="focusPolicy">
     <enum>Qt::WheelFocus</enum>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="tristate">
     <bool>false</bool>
    </property>
   </widget>
   <widget class="QCheckBox" name="ad_checkBox">
    <property name="geometry">
     <rect>
      <x>60</x>
      <y>480</y>
      <width>61</width>
      <height>71</height>
     </rect>
    </property>
    <property name="sizePolicy">
     <sizepolicy hsizetype="Minimum" vsizetype="Fixed">
      <horstretch>0</horstretch>
      <verstretch>0</verstretch>
     </sizepolicy>
    </property>
    <property name="font">
     <font>
      <pointsize>14</pointsize>
     </font>
    </property>
    <property name="focusPolicy">
     <enum>Qt::WheelFocus</enum>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="tristate">
     <bool>false</bool>
    </property>
   </widget>
   <widget class="QCheckBox" name="bbed_checkBox">
    <property name="geometry">
     <rect>
      <x>440</x>
      <y>700</y>
      <width>61</width>
      <height>71</height>
     </rect>
    </property>
    <property name="sizePolicy">
     <sizepolicy hsizetype="Minimum" vsizetype="Fixed">
      <horstretch>0</horstretch>
      <verstretch>0</verstretch>
     </sizepolicy>
    </property>
    <property name="font">
     <font>
      <pointsize>14</pointsize>
     </font>
    </property>
    <property name="focusPolicy">
     <enum>Qt::WheelFocus</enum>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="tristate">
     <bool>false</bool>
    </property>
   </widget>
   <widget class="Line" name="line_2">
    <property name="geometry">
     <rect>
      <x>-30</x>
      <y>630</y>
      <width>1201</width>
      <height>20</height>
     </rect>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Plain</enum>
    </property>
    <property name="lineWidth">
     <number>3</number>
    </property>
    <property name="orientation">
     <enum>Qt::Horizontal</enum>
    </property>
   </widget>
   <widget class="Line" name="line_3">
    <property name="geometry">
     <rect>
      <x>360</x>
      <y>190</y>
      <width>20</width>
      <height>441</height>
     </rect>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Plain</enum>
    </property>
    <property name="lineWidth">
     <number>3</number>
    </property>
    <property name="orientation">
     <enum>Qt::Vertical</enum>
    </property>
   </widget>
   <widget class="Line" name="line_4">
    <property name="geometry">
     <rect>
      <x>740</x>
      <y>190</y>
      <width>20</width>
      <height>441</height>
     </rect>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Plain</enum>
    </property>
    <property name="lineWidth">
     <number>3</number>
    </property>
    <property name="orientation">
     <enum>Qt::Vertical</enum>
    </property>
   </widget>
   <widget class="QPushButton" name="bb_pushButton">
    <property name="enabled">
     <bool>false</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>90</x>
      <y>330</y>
      <width>191</width>
      <height>71</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>16</pointsize>
     </font>
    </property>
    <property name="autoFillBackground">
     <bool>false</bool>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(255, 198, 199);</string>
    </property>
    <property name="text">
     <string>ก่อน อาหาร</string>
    </property>
   </widget>
   <widget class="QPushButton" name="ab_pushButton">
    <property name="enabled">
     <bool>false</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>90</x>
      <y>480</y>
      <width>191</width>
      <height>71</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>16</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(255, 232, 194);</string>
    </property>
    <property name="text">
     <string>หลัง อาหาร</string>
    </property>
   </widget>
   <widget class="QPushButton" name="bl_pushButton">
    <property name="enabled">
     <bool>false</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>480</x>
      <y>330</y>
      <width>191</width>
      <height>71</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>16</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(255, 254, 202);</string>
    </property>
    <property name="text">
     <string>ก่อน อาหาร</string>
    </property>
   </widget>
   <widget class="QPushButton" name="al_pushButton">
    <property name="enabled">
     <bool>false</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>480</x>
      <y>480</y>
      <width>191</width>
      <height>71</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>16</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(219, 255, 199);</string>
    </property>
    <property name="text">
     <string>หลัง อาหาร</string>
    </property>
   </widget>
   <widget class="QPushButton" name="bd_pushButton">
    <property name="enabled">
     <bool>false</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>860</x>
      <y>330</y>
      <width>191</width>
      <height>71</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>16</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(185, 227, 255);</string>
    </property>
    <property name="text">
     <string>ก่อน อาหาร</string>
    </property>
   </widget>
   <widget class="QPushButton" name="ad_pushButton">
    <property name="enabled">
     <bool>false</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>860</x>
      <y>480</y>
      <width>191</width>
      <height>71</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>16</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(201, 205, 255);</string>
    </property>
    <property name="text">
     <string>หลัง อาหาร</string>
    </property>
   </widget>
   <widget class="QPushButton" name="bbed_pushButton">
    <property name="enabled">
     <bool>false</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>480</x>
      <y>700</y>
      <width>191</width>
      <height>71</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>16</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(250, 211, 255);</string>
    </property>
    <property name="text">
     <string>ก่อน อาหาร</string>
    </property>
   </widget>
   <widget class="QLabel" name="b_label">
    <property name="geometry">
     <rect>
      <x>120</x>
      <y>240</y>
      <width>131</width>
      <height>41</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>20</pointsize>
     </font>
    </property>
    <property name="text">
     <string>มื้อเช้า</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="a_label">
    <property name="geometry">
     <rect>
      <x>500</x>
      <y>240</y>
      <width>131</width>
      <height>41</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>20</pointsize>
     </font>
    </property>
    <property name="text">
     <string>มื้อเที่ยง</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="label_3">
    <property name="geometry">
     <rect>
      <x>880</x>
      <y>240</y>
      <width>131</width>
      <height>41</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>20</pointsize>
     </font>
    </property>
    <property name="text">
     <string>มื้อเย็น</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QPushButton" name="back_pushButton">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>80</y>
      <width>131</width>
      <height>41</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">color: rgb(255, 255, 255);
background-color: rgb(166, 0, 0)</string>
    </property>
    <property name="text">
     <string>ย้อนกลับ</string>
    </property>
   </widget>
   <widget class="QLabel" name="bb_cr_label">
    <property name="enabled">
     <bool>false</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>80</x>
      <y>290</y>
      <width>31</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="pixmap">
     <pixmap resource="../Resources/resources.qrc">:/icons/Flat_tick_icon.svg.png</pixmap>
    </property>
    <property name="scaledContents">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QLabel" name="ab_cr_label">
    <property name="enabled">
     <bool>false</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>80</x>
      <y>440</y>
      <width>31</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="pixmap">
     <pixmap resource="../Resources/resources.qrc">:/icons/Flat_tick_icon.svg.png</pixmap>
    </property>
    <property name="scaledContents">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QLabel" name="bl_cr_label">
    <property name="enabled">
     <bool>false</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>470</x>
      <y>290</y>
      <width>31</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="pixmap">
     <pixmap resource="../Resources/resources.qrc">:/icons/Flat_tick_icon.svg.png</pixmap>
    </property>
    <property name="scaledContents">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QLabel" name="al_cr_label">
    <property name="enabled">
     <bool>false</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>470</x>
      <y>440</y>
      <width>31</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="pixmap">
     <pixmap resource="../Resources/resources.qrc">:/icons/Flat_tick_icon.svg.png</pixmap>
    </property>
    <property name="scaledContents">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QLabel" name="bd_cr_label">
    <property name="enabled">
     <bool>false</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>850</x>
      <y>290</y>
      <width>31</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="pixmap">
     <pixmap resource="../Resources/resources.qrc">:/icons/Flat_tick_icon.svg.png</pixmap>
    </property>
    <property name="scaledContents">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QLabel" name="ad_cr_label">
    <property name="enabled">
     <bool>false</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>850</x>
      <y>440</y>
      <width>31</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="pixmap">
     <pixmap resource="../Resources/resources.qrc">:/icons/Flat_tick_icon.svg.png</pixmap>
    </property>
    <property name="scaledContents">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QLabel" name="bbed_cr_label">
    <property name="enabled">
     <bool>false</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>470</x>
      <y>660</y>
      <width>31</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="pixmap">
     <pixmap resource="../Resources/resources.qrc">:/icons/Flat_tick_icon.svg.png</pixmap>
    </property>
    <property name="scaledContents">
     <bool>true</bool>
    </property>
   </widget>
  </widget>
 </widget>
 <resources>
  <include location="../Resources/resources.qrc"/>
 </resources>
 <connections>
  <connection>
   <sender>bb_checkBox</sender>
   <signal>clicked(bool)</signal>
   <receiver>bb_pushButton</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>77</x>
     <y>368</y>
    </hint>
    <hint type="destinationlabel">
     <x>162</x>
     <y>354</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>ad_checkBox</sender>
   <signal>clicked(bool)</signal>
   <receiver>ab_pushButton</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>73</x>
     <y>511</y>
    </hint>
    <hint type="destinationlabel">
     <x>186</x>
     <y>507</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>bd_checkBox</sender>
   <signal>clicked(bool)</signal>
   <receiver>bl_pushButton</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>451</x>
     <y>368</y>
    </hint>
    <hint type="destinationlabel">
     <x>585</x>
     <y>359</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>al_checkBox</sender>
   <signal>clicked(bool)</signal>
   <receiver>al_pushButton</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>460</x>
     <y>516</y>
    </hint>
    <hint type="destinationlabel">
     <x>570</x>
     <y>516</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>bl_checkBox</sender>
   <signal>clicked(bool)</signal>
   <receiver>bd_pushButton</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>833</x>
     <y>365</y>
    </hint>
    <hint type="destinationlabel">
     <x>940</x>
     <y>373</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>ab_checkBox</sender>
   <signal>clicked(bool)</signal>
   <receiver>ad_pushButton</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>820</x>
     <y>500</y>
    </hint>
    <hint type="destinationlabel">
     <x>938</x>
     <y>513</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>bbed_checkBox</sender>
   <signal>clicked(bool)</signal>
   <receiver>bbed_pushButton</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>446</x>
     <y>726</y>
    </hint>
    <hint type="destinationlabel">
     <x>575</x>
     <y>723</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>bb_checkBox</sender>
   <signal>clicked(bool)</signal>
   <receiver>bb_cr_label</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>78</x>
     <y>365</y>
    </hint>
    <hint type="destinationlabel">
     <x>103</x>
     <y>300</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>bd_checkBox</sender>
   <signal>clicked(bool)</signal>
   <receiver>bl_cr_label</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>464</x>
     <y>376</y>
    </hint>
    <hint type="destinationlabel">
     <x>486</x>
     <y>306</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>bl_checkBox</sender>
   <signal>clicked(bool)</signal>
   <receiver>bd_cr_label</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>832</x>
     <y>367</y>
    </hint>
    <hint type="destinationlabel">
     <x>868</x>
     <y>303</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>ad_checkBox</sender>
   <signal>clicked(bool)</signal>
   <receiver>ab_cr_label</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>67</x>
     <y>521</y>
    </hint>
    <hint type="destinationlabel">
     <x>100</x>
     <y>459</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>al_checkBox</sender>
   <signal>clicked(bool)</signal>
   <receiver>al_cr_label</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>458</x>
     <y>525</y>
    </hint>
    <hint type="destinationlabel">
     <x>497</x>
     <y>449</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>ab_checkBox</sender>
   <signal>clicked(bool)</signal>
   <receiver>ad_cr_label</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>836</x>
     <y>523</y>
    </hint>
    <hint type="destinationlabel">
     <x>869</x>
     <y>457</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>bbed_checkBox</sender>
   <signal>clicked(bool)</signal>
   <receiver>bbed_cr_label</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>468</x>
     <y>742</y>
    </hint>
    <hint type="destinationlabel">
     <x>492</x>
     <y>679</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
